mlir/include/mlir/Dialect/LLVMIR/GENXOps.td

```diff
@@ -110,37 +110,37 @@ class GENX_DeviceFunctionOp<string mnemonic, string device_function,
 // Thread index and Block index
 
 def GENX_ThreadIdXOp : GENX_DeviceFunctionOp<"workitem.id.x",
-                        "get_local_id", 0>;
+                        "_Z12get_local_idj", 0>;
```

Comment:
I wish the translator had a C interface for the API it exposes... However it does not matter because this is temporary. We will use the IGC builtins to lower these operations, correct ?

Reply:
Yes, we will move to IGC builtins.